HiP-UserStore/Controllers/PhotoController.cs

```diff
@@ -0,0 +1,156 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using PaderbornUniversity.SILab.Hip.UserStore.Core.ReadModel;
+using PaderbornUniversity.SILab.Hip.UserStore.Utility;
+using PaderbornUniversity.SILab.Hip.UserStore.Core;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using PaderbornUniversity.SILab.Hip.EventSourcing;
+using PaderbornUniversity.SILab.Hip.UserStore.Core.WriteModel;
+using PaderbornUniversity.SILab.Hip.UserStore.Model.Events;
+using PaderbornUniversity.SILab.Hip.UserStore.Model;
+using Microsoft.AspNetCore.StaticFiles;
+using MongoDB.Driver;
+using PaderbornUniversity.SILab.Hip.UserStore.Model.Entity;
+using System.ComponentModel.DataAnnotations;
+
+namespace PaderbornUniversity.SILab.Hip.UserStore.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    public class PhotoController : Controller
```

Comment:
* POST- & PUT-methods should be combined into only 1 PUT-method
* We probably need a DELETE-method as well